packages/core/src/customizations/fragments/ClientSearchSuggestions.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from '@faststore/graphql-utils'
+
+export const fragment = gql`
+  fragment ClientSearchSuggestions on Query {
+    search(first: 5, term: $term, selectedFacets: $selectedFacets) {
```

Comment:
Allowing to customize the amount of items will surely be a task in the future. Not sure if we should do something about it now, tho.

Reply:
this is just the default behavior. So when people use the starter they will override this file. But we can use a variable instead, actually.